uiowa/uiowa
# Add schema to dentistry.uiowa.edu.

Resolves #4600 

# How to test

`blt ds --site=dentistry.uiowa.edu`
`drush @dentistry.local config:import --source ../config/sites/dentistry.uiowa.edu --partial`
`drush @dentistry.local cim`

See this comment for schema specifics: https://github.com/uiowa/uiowa/issues/4600#issuecomment-1013752236

Load the homepage, check for the schema. See that the separate Dentist schema is not provided.
Load https://dentistry.uiowa.ddev.site/patient-care and see that the Dentist schema (full) is provided.
